pkg/oracles/contracts/StableLPOracle.sol

```diff
@@ -42,13 +57,12 @@ contract StableLPOracle is LPOracleBase {
 
     /// @inheritdoc LPOracleBase
     function _computeTVL(int256[] memory prices) internal view override returns (uint256 tvl) {
```

Comment:
This is outside of the diff, but we should document in `LPOracleBase` that `prices` is a FP-18 (min price would mean different things if `prices` wasn't normalized to 18 decimals).